src/app/components/LinkedData/__snapshots__/index.test.jsx.snap

```diff
@@ -1,40 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`LinkData should correctly render metadata for frontpages 1`] = `
-<HelmetWrapper
-  defer={true}
-  encodeSpecialCharacters={true}
->
-  <script
-    type="application/ld+json"
-  >
-    {"@context":"http://schema.org","@type":"WebPage","url":"https://www.bbc.com/igbo","publisher":{"@type":"NewsMediaOrganization","name":"BBC News","publishingPrinciples":"https://www.bbc.com/news/help-41670342","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/igbo.png"}},"image":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/igbo.png"},"thumbnailUrl":"https://news.files.bbci.co.uk/ws/img/logos/og/igbo.png","mainEntityOfPage":{"@type":"WebPage","@id":"https://www.bbc.com/igbo","name":"Ogbako"}}
-  </script>
-</HelmetWrapper>
-`;
+exports[`LinkData should correctly render metadata for frontpages 1`] = `<div />`;
```

Comment:
Once PR #3694 is merged, snapshot coverage might be fixed.

Reply:
We should block this PR on that then :), lots of lost coverage